pkg/config/log_nix.go

```diff
@@ -16,6 +16,10 @@ func GetSyslogURI() string {
 		if uri == "" {
 			uri = defaultSyslogURI
 		}
+	} else {
```

Comment:
The logic in this function is a bit complicated, could it just be:
```go
func GetSyslogURI() string {
	enabled := Datadog.GetBool("log_to_syslog")
	uri := Datadog.GetString("syslog_uri")

    if !enabled {
        return ""
    }
    if uri == "" {
        return defaultSyslogURI
    }
    return uri
}
```
Seems clearer.

Reply:
Absolutely!